packages/phoenix/src/lib.rs

```diff
@@ -1,3 +1,7 @@
 #![no_std]
 
 pub mod utils;
+
+pub mod constants {
+    pub const LP_TOKEN_DECIMALS: u32 = 10;
+}
```

Comment:
Remove it.
Add it as a parameter for Factory initialization under liquidity pools params.